src/org/labkey/targetedms/chart/ChromatogramDataset.java

```diff
@@ -1155,9 +1206,8 @@ public int compare(TransChromInfoPlusTransition t1, TransChromInfoPlusTransition
         }
     }
 
-    static class MoleculeTransChromInfoPlusTransition
+    static class MoleculeTransChromInfoPlusTransition extends TransitionChromInfoPlusGeneralTransition<MoleculeTransition>
     {
-        private TransitionChromInfo _transChromInfo;
         private MoleculeTransition _transition;
```

Comment:
Could push this member variable up to the abstract superclass and make getTransition() non-abstract there.